pages/product/index.js

```diff
@@ -0,0 +1,32 @@
+let filterrOpen = document.querySelector(".filter-open");
```

Comment:
This and code below actually do same work, could you move it to function and then reuse 2 times with different arguments?